src/components/Transfer/AddToMetamask.tsx

```diff
@@ -106,7 +81,7 @@ export default function AddToMetamask() {
       variant="outlined"
       className={classes.addButton}
     >
-      {addToWalletText({ isMetaMask, isBraveWallet, isKuCoinWallet })}
+      {`Add to ${wallet?.getName()}`}
```

Comment:
We can add a fallback in case `getName` function for some reason does not exist or return falsy value.

```suggestion
      {`Add to ${wallet?.getName()} || wallet`}
```